src/components/core/DpTableCardList/DpTableCard.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta } from '@storybook/addon-docs'
+import DpTableCard from './DpTableCard'
+
+<Meta
+    title="Components/TableCard"
+    component={DpTableCard}
+/>
+
+# TableCard
+
+The TableCard component is used to display tabular data in a visually organised and structured way.
+
+It's often used to present data in a list, making it easier for users to scan and understand the information.
+
+## Custom usage
+
+```html
+<div>
+    <ul class="o-list o-list--card u-mb">
+      <div class="o-list__item">
```

Comment:
the component itself resolves in a li. so the parent should be a ul.
Do we need the div here?

Reply:
thanks
Done in https://github.com/demos-europe/demosplan-ui/pull/548/commits/0a8415e5b3023fe483afc217ae0e6ab54cfaf3af